pages/index.vue

```diff
@@ -1342,8 +1346,6 @@ export default {
   width 100%
   z-index 10
   backdrop-filter blur(30px)
```

Comment:
I'd like to ditch this effect, as it's not doing us any favors for performance. This should do the trick, but only when the nav is not "frozen" by headroom:
```
.section-nav:not(.headroom--frozen)
  background linear-gradient(180deg, rgba(0,0,0,0.96) 50%, rgba(0,0,0,0.7))
```